drivers/bridge/bridge.go

```diff
@@ -387,6 +371,14 @@ func (d *driver) configure(option map[string]interface{}) error {
 	}
 
 	if config.EnableIPTables {
+		if _, err := os.Stat("/proc/sys/net/bridge"); err != nil {
+			if out, err := exec.Command("modprobe", "-va", "bridge", "br_netfilter").CombinedOutput(); err != nil {
+				logrus.Warnf("Running modprobe bridge br_netfilter failed with message: %s, error: %v", out, err)
+			}
+		}
+		if err := iptables.FirewalldInit(); err != nil {
```

Comment:
By moving FirewalldInit() under `EnableIPTables`, this might impact other drivers inadvertently that uses IPTables for its operation (such as Overlay in Kernel < 3.16).
This should be moved to iptables package. WDYT ?